.gitignore

```diff
@@ -1,2 +1,3 @@
 */*.json
+image_info.json
```

Comment:
You probably don't need or want this any more. (And wasn't it yaml anyway, not json?)

Reply:
it was both, but good catch, I'll remove it.